addContentHomePage.js

```diff
@@ -38,12 +39,41 @@ var calculateWorkedHoursByProject = function (){
   }
   var weekTotalReference = document.querySelector('.weekly-pane-footer-total').querySelector('p');
   for( var i of Object.keys(map)){
+      var valueString = map[i].toString();
+      map[i] = isHourly() ? valueString.split(".")[0] + "h " + decimalStringToMinutes(valueString) + "m" : map[i];
       weekTotalReference.insertAdjacentHTML('beforebegin', 
           `<p class="mr-4 mb-0 tasksSpan">${i} &nbsp;<strong>${map[i]}</strong></p>`);
   }
   var map = {};
 };
 
+var extractTime = function(time){
+  var hours = 0;
+  var minutes = 0;
+  if (time.includes("h")){
+      prefix = time.split("h");
+      hours = parseInt(prefix[0]);
+      time = prefix.length > 1 ? prefix[1].trim() : time;
+  }
+
+  if (time.includes("m")){
+    time = parseInt(time.split("m")[0]);
+    minutes += (time/60);
+  }
+
+  return Math.round((hours + minutes + Number.EPSILON) * 100) / 100;
+  
+};
+
+var isHourly = function(){
+  var footerHours = document.querySelectorAll(".mr-4.mb-0 strong");
+  return footerHours[footerHours.length - 1].textContent.match('([A-Za-z])');
+};
+
+var decimalStringToMinutes = function(valueString){
+  return Math.round((parseInt(valueString.split(".")[1].substring(0,2))/100) * 60);
```

Comment:
Apparently, when we have hours with no minutes, this line breaks because it can find this reference to subString 
`valueString.split(".")[1]`

if you want to do something like this so we avoid "null pointers"
`var containMinutes = valueString.split(".")[1];
  return containMinutes ? Math.round((parseInt(valueString.split(".")[1].substring(0,2))/100) * 60) : 0;`